plugins/performance-lab/load.php

```diff
@@ -83,14 +83,15 @@ function perflab_render_generator(): void {
  *
  * @since 3.0.0
  *
- * @return array<string, array{'constant': string, 'experimental'?: bool}> Associative array of $plugin_slug => $plugin_data pairs.
+ * @return array<string, array{'constant': string, 'experimental'?: bool, 'suggest'?: string[]}> Associative array of $plugin_slug => $plugin_data pairs.
```

Comment:
I'm not sure we want to establish this as something we may want to do more, given that this is not a thing that WordPress Core supports.

Therefore instead of introducing a "schema" change here, I think we should for now hard-code this in `perflab_install_and_activate_plugin()`. Feels more hacky, but is more future-proof as it doesn't change any portion of the API. Given that this PR simply takes the "suggested" feature as if it was "required", that part would need more thought if we truly wanted to establish a way to _suggest_ a feature.